### DISCUSSION TITLE
The sound isnt good enough.

### DISCUSSION THREAD

**Discussion by mawess** [jamulussoftware/jamulus#1822] [Ideas]
current sound quality setting  "high" is still compressed and introduces sound artifacts.

please add a sound quality setting that wont use  compression at all. 

lot of servers will manage and benefit from that.

**Comment by mawess** [comment]
why is this moved from issues? it is an issue: there are a lot of aliasing noise from frequencies far lower than are expected.

**Reply by gilgongo** [reply]
We're trying to keep issues to be a list of actionable specifications for work that have been discussed and agreed, or clearly specified bugs.

**Comment by mawess** [comment]
ok. my bad.

**Comment by jujudusud** [comment]
Could you please tell use more on your use case:
- your hardware
- the setup you made (do you run on 128 or 64 - little network packets etc)
- your internet (fiber etc ...)
- how does the jitters live in your installation ...

There is a lot of things that can make the sound worse.
Try to use 128 first if you are not sure that your internet line is sufficiently big and stable.

Sound quality has to be put in balance with latency reachable.

**Reply by jujudusud** [reply]
Any news here ?
